Artesian/Artesian.SDK/Factory/MarketData.cs

```diff
@@ -222,5 +225,53 @@ public IBidAskWritable EditBidAsk()
             var bask = new BidAsk(this);
             return bask;
         }
+
+        /// <summary>
+        /// Update Derived Configuration
+        /// </summary>
+        /// <param name="derivedCfg">The Derived Configuration Muv to be updated</param>
+        /// <param name="force">Force the update of configuration also if another rebuild process is running (Defualt=false)</param>
+        /// <param name="ctk">Cancellation Token</param>
+        /// <returns></returns>
+        public async Task UpdateDerivedConfiguration(DerivedCfgMuv derivedCfg, bool force, CancellationToken ctk = default)
```

Comment:
force is optional, defaults to false in backend.

keep the same here.

use Overloads for DX over ctk propagation.